src/main/resources/lang/en.yml

```diff
@@ -723,6 +723,16 @@ item:
       running: "<green>Running"
       idle: "<yellow>Idle"
 
+  fireproof_rune:
+    name: "Fireproof Rune"
+    lore: |-
+      <arrow> Drop your weapon and rune, and pray for 800 hours
+      <arrow> to earn the blessing of the Fire God
+      <arrow> so that your weapon become fireproof.
```

Comment:
We have an automatic line wrapping system so you don't need to break the line up into multiple